packages/browser-wallet/src/shared/utils/basicHelpers.ts

```diff
@@ -0,0 +1,14 @@
+export function isDefined<T>(v?: T): v is T {
+    return v !== undefined;
+}
+
+export const notNull = <T>(v: T | null | undefined): v is T => v != null;
+
+export function noOp(): void {
+    return undefined;
+}
```

Comment:
:thinking: 
We might need some type-checking on this project?

Reply:
As far as I understand, undefined != void

Reply:
what do you mean we might need type checking? I agree that it doesn't make sense, but apparently typescript accepts it :shrug: 

I'll correct it to an empty block instead.